plugins/modules/identity/keycloak/keycloak_user_rolemapping.py

```diff
@@ -0,0 +1,399 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Comment:
You need to add a copyright line, like this. Also the remainder of the license header needs to be adjusted.
```suggestion
# Copyright (c) 2022, ...
# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
# SPDX-License-Identifier: GPL-3.0-or-later
```